src/main/scala/org/deepdive/ddlog/DeepDiveLogParser.scala

```diff
@@ -252,7 +252,7 @@ class DeepDiveLogParser extends JavaTokenParsers {
 
   def relationType: Parser[RelationType] =
     ( "like" ~> relationName ^^ { RelationTypeAlias(_) }
-    | rep1sep(columnDeclaration, ",") ^^ {
+    | "(" ~> rep1sep(columnDeclaration, ",") <~ ")" ^^ {
```

Comment:
Can we also add a token `"rows" ~> "like"` to the other case? (#36)

It seems `relationType` parser is now used only by function declaration.  Why don't we rename it to `functionInputOutputType` or so?